moonsla.go

```diff
@@ -164,11 +164,12 @@ func main() {
 	if !ok {
 		fmt.Println("Please set your SLACK_TOKEN")
 	}
-	api := slack.New(slackToken)
-
 	logger := log.New(os.Stdout, "slack-bot: ", log.Lshortfile|log.LstdFlags)
-	slack.SetLogger(logger)
-	api.SetDebug(false)
+
+	api := slack.New(
+		slackToken,
+		slack.OptionDebug(true),
```

Comment:
Could you switch this back to the default of false? Making it configurable would also be ok. 

With this enabled I'm getting spammed by logs:

```
slack-bot: 2019/10/03 08:39:10 websocket_managed_conn.go:326: Sending PING  1
slack-bot: 2019/10/03 08:39:10 websocket_managed_conn.go:369: Incoming Event: {"type":"pong","timestamp":1570084750,"reply_to":1}
slack-bot: 2019/10/03 08:39:40 websocket_managed_conn.go:326: Sending PING  2
slack-bot: 2019/10/03 08:39:40 websocket_managed_conn.go:369: Incoming Event: {"type":"pong","timestamp":1570084780,"reply_to":2}

```

```suggestion
		slack.OptionDebug(false),
```

Reply:
Oops! Sorry. Fixed in 4d179b2